docs/reference/components/sources/datadog_logs.cue

```diff
@@ -0,0 +1,115 @@
+package metadata
+
+components: sources: datadog_logs: {
+	_port: 8080
+
+	title: "Heroku Logplex"
+
+	description: """
+		Receives logs from a local Datadog Agent.
+		"""
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["aggregator"]
```

Comment:
It seems like this could be an `aggregator` or a `sidecar` alongside a dd-agent.

Reply:
Indeed, will add.